### PR TITLE
Sp2 Front End Campaign Filters

### PR DESCRIPTION
**Pull request by tkiethanom**
r? @rlr @tspurway 

Added UI to handle latest filters (past, scheduled and in_flight) added to Campaigns.

### CODE REVIEW

File: front_end/src/components/Campaigns/CampaignList/CampaignRow.js [hunk 1]
```diff
@@ -1,28 +1,52 @@
...
+    countries.map((row, index) => {
+      if (index > 0) {
+        output += ', ';
+      }
+
+      country = getCountry(row, this.props.init_countries);
+      if (country !== undefined && country.country_name !== undefined) {
+        output += country.country_name;
+      }
+    });
```
Comment (rlr) [inline front_end/src/components/Campaigns/CampaignList/CampaignRow.js]: This is a little messy and it looks like there are cases where you could end up with `country1, , country2`. I would build an array of country names then:

```
>> ['x', 'y', 'z'].join(', ')
"x, y, z"
```

**Comment by rlr** [comment]
Other than that, this looks good to me.

**Comment by tkiethanom** [comment]
Updated this to use array.join()

**Comment by tkiethanom** [comment]
Commented out Approvals fixes #176

**Comment by rlr** [comment]
:+1: r+

**Pull request merged**